api/v1/controllers/collectors.js

```diff
@@ -270,12 +271,16 @@ function heartbeat(req, res, next) {
       throw new apiErrors.ForbiddenError({
         explanation: 'Authentication Failed',
       });
-    } else if (o.status !== 'Running' && o.status !== 'Paused' && o.status !== 'MissedHeartbeat') {
+    } else if (o.status === status.Stopped) {
```

Comment:
Do we really want to accept heartbeats from paused collectors? What's the use case here?